docs/architecture/payments.rst

```diff
@@ -1,6 +1,48 @@
 Payments
 ========
 
+Supported Gateways
+------------------
+
+The default configuration uses the *dummy* backend.
+It's meant to allow developers to easily simulate different payment results.
+
+Here is a list of supported payment providers:
```

Comment:
Could you put this under a sub section instead? So we can easily insert a third party payment provider section later on.